core/api/serializers/country_programme.py

```diff
@@ -35,25 +40,92 @@ class Meta:
 
 
 class CPRecordSerializer(serializers.ModelSerializer):
-    blend = serializers.StringRelatedField()
-    substance = serializers.StringRelatedField()
+    chemical_name = serializers.SerializerMethodField()
+    chemical_group_name = serializers.SerializerMethodField()
     record_usages = CPUsageSerializer(many=True)
 
     class Meta:
         model = CPRecord
         fields = [
             "id",
-            "blend",
-            "substance",
-            "country_programme_report_id",
+            "chemical_name",
+            "chemical_group_name",
             "display_name",
             "section",
             "imports",
```

Comment:
Are `imports`, `exports` and `production`, `usage.quantity` serialized as strings on purpose?

Reply:
Decimals need to be serialised as strings in JSON, since float representation would lose precision.